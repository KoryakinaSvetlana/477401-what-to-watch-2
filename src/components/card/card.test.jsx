import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter as Router} from "react-router-dom";
import Card from "./card";

it(`Card correctly renders`, () => {
  const tree = renderer.create(
      <Router>
        <Card
          film={{id: 1, name: `1`}}
          onHover={() => {}}
        />
      </Router>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
