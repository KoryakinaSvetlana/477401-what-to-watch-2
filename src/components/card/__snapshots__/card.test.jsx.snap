// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card correctly renders 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="1"
      height="175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      href="films/1"
      onClick={[Function]}
    >
      1
    </a>
  </h3>
</article>
`;
